titiler/core/setup.py

```diff
@@ -6,15 +6,14 @@
     long_description = f.read()
 
 inst_reqs = [
-    "fastapi==0.63.0",
+    "fastapi==0.65.1",
     "geojson-pydantic",
     "jinja2>=2.11.2,<3.0.0",
     "morecantile>=2.1,<2.2",
     "numpy",
     "pydantic",
     "rasterio",
-    "rio-cogeo>=2.1,<2.2",
-    "rio-tiler>=2.0.5,<2.1",
+    "rio-tiler>=2.1,<2.2",
```

Comment:
should we pin rio-tiler ?

Reply:
Likewise, IMO a range is ideal for `titiler.core`.